src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -190,17 +193,25 @@ private void initPanel() {
         selectDocument.setOnAction(e -> ooBase.guiActionSelectDocument(false));
 
         setStyleFile.setMaxWidth(Double.MAX_VALUE);
-        setStyleFile.setOnAction(event ->
-                dialogService.showCustomDialogAndWait(new StyleSelectDialogView(loader))
-                             .ifPresent(selectedStyle -> {
-                                 style = selectedStyle;
-                                 try {
-                                     style.ensureUpToDate();
-                                 } catch (IOException e) {
-                                     LOGGER.warn("Unable to reload style file '" + style.getPath() + "'", e);
-                                 }
-                                 dialogService.notify(Localization.lang("Current style is '%0'", style.getName()));
-                             }));
+        setStyleFile.setOnAction(event -> {
+            StyleSelectDialogView styleDialog = new StyleSelectDialogView(loader);
+            dialogService.showCustomDialogAndWait(styleDialog)
+                         .ifPresent(selectedStyle -> {
+                             jStyle = selectedStyle;
+                             currentStyleType = styleDialog.getSelectedStyleType();
+                             try {
+                                 jStyle.ensureUpToDate();
+                             } catch (IOException e) {
+                                 LOGGER.warn("Unable to reload style file '" + jStyle.getPath() + "'", e);
```

Comment:
```suggestion
                                 LOGGER.warn("Unable to reload style file  '{} '", e, jstyle.getPath() );
```
intelli should give you a hint, Parameterized logging https://www.slf4j.org/faq.html#logging_performance

Reply:
Done